demo/summer-2016.html

```diff
@@ -211,11 +211,9 @@
 <script src="kitchen-sink/require.js"></script>
 <script>
 require.config({ paths : { "ace" : "../lib/ace" } });
-require(["ace/ace", "ace/ext/language_tools", "ace/ext/katex"], function(ace, langTools, katex) {
+require(["ace/ace", "ace/ext/language_tools", "ace/ext/katex", "ace/ext/papeeria/katex-drawer"], function(ace, langTools, katex, katexDrawer) {
```

Comment:
is it okay that file is called `katex-previewer` while module is called `katex-drawer` ?